src/integrationTest/sam.test.ts

```diff
@@ -343,10 +331,33 @@ describe('SAM Integration Tests', async () => {
                         )
                     })
 
-                    await vscode.commands.executeCommand<any>(
-                        codeLens.command!.command,
-                        ...codeLens.command!.arguments!
-                    )
+                    const rootUri = vscode.Uri.file(appPath)
+                    const launchConfig = new LaunchConfiguration(rootUri)
+                    const testConfig = {
+                        type: 'aws-sam',
+                        request: 'direct-invoke',
+                        name: 'test-config-1',
```

Comment:
Within `debugSessionStartedAndStoppedPromise`, is it possible to check for this `name` as the expected session name?

Reply:
My thought is that if the name is unique for each test, checking name is a cheap way of checking "did the debug session that I'm interested in start/end?"

Reply:
As discussed, this will be addressed in a followup issue/PR.